### ISSUE TITLE
Request - quad rules (and cleanup)

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#8491]
Help us understand your request (check below):

- [x] other

**Describe what you're trying to do** 

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctquad is messier than it needs to be, and the rules could use some polish.

The last 'word' in the data includes

```



 1:100000       |     2
 1:100,000      |     7
 1:125000       |     4
 1:125,000      |     5
 1:250,000      |    21
 1:50,000       |     1
 1:62500        |     2
 1:62,500       |     1
 1:625,000      |     1
 7.5            |     3
 A1             |     1




 F/10           |     1
 F/7            |     1


 H/SE           |     1


 m              |     5


 mi             |     5


 min            |    55
 minute         |  2796


 NE             |     1

 NW             |     1


 sheet          |     1
```

I propose we ban commas, they tend to find ways to be annoying. Alternatively we could require them, maybe the readability overrides any theoretical nuisances, as long as things are consistent. the existence of "1:625,000"  suggests they don't work for that though - and that should lose a zero.

I propose we standardize that 'bare' terms are 1:250K - that's the origin of these data and should be *mostly* correct. Alternatively we could add that to be explicit, but I suspect that's not what the oldest users want - I just want to be consistent/not redundant and don't care what form that takes.

Billings 1:250,000 sheet [[ link ]](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctquad#billings_1_250_000_sheet)

should be "Billings" if we go there, drop the 'sheet' in any case.

The various m-things should be updated to 'minute'

These

 Grandmother Mountain East 7.5
 Grandmother Mountain West 7.5
 Arroyo de las Calabacillas 7.5

should have 'minute' appended

The 'A1' are all just nonsense?? Eagle A1 1:250,000 [[ link ]](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctquad#eagle_a1_1_250_000) links to a 1:63-6 map as a reference....

IDK what Comox 92 F/10 [[ link ]](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctquad#comox_92_f_10) is, probably nonsense, suggest delete and leave a remark where necessary

Horne Lake 92 F/7 [[ link ]](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctquad#horne_lake_92_f_7) - ???? delete

 Princeton 92 H/SE [[ link ]](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctquad#princeton_92_h_se) ????? delete

Maybe all of the direction-containing terms are nonsense?? None seem to have useful authorities, anyway.

I'll assume I'm not lost and confused and all of this should be done in some way if nobody shows up to set me straight before the next time I stumble upon this issue....

HELP!

**Comment by mkoo** [comment]
Adding to the Geography committee discussion list. If you have strong opinions on this join the next meeting

**Comment by WaigePilson** [comment]
@dustymc 

> Comox 92 F/10 [[ link ]](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctquad#comox_92_f_10) is, probably nonsense, suggest delete and leave a remark where necessary

Horne Lake 92 F/7 [[ link ]](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctquad#horne_lake_92_f_7) - ???? delete

Princeton 92 H/SE [[ link ]](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctquad#princeton_92_h_se) ????? delete

All of the above are Canadian National Topographic System (NTS) maps (https://natural-resources.canada.ca/maps-tools-publications/maps/topographic-maps). For example "Horne Lake" is the map name, and "92 F/7" refers to the particular grid area in British Columbia of this map (just north of the border with Washington around 117deg longitude). It's a very similar system to the USA, but just a different grid. As long as the quad name has this grid reference (e.g., "92 F/7") and the documentation lists the country and state (e.g., "British Columbia, Canada") in it I would STRONGLY advocate that we keep these in the quad code table. Otherwise, we are restricting that these quads only can be used for U.S. maps?

**Comment by dustymc** [comment]
> restricting that these quads only can be used for U.S. maps

Not at all (at least I wouldn't favor that),  just trying to clean up a bunch of maybe-nonsense (or not, IDK!) data that got loaded without sufficient documentation (or I'd be able to tell if it was nonsense...) and establish guidelines going forward.

I'm more sure than ever that good documentation is critical for all "authority" values, no surprise there.

I'm sure that we need some standardization (one way to say ``minute`` or ``100000``), but don't much care about the details.

Possibly we could move 1:250K back to https://handbook.arctosdb.org/documentation/higher-geography.html, or maybe develop some default/rule/documentation/whatever here, I'm totally up for anything on that front.

I think I deleted some unused and undocumented stuff (should periodically do that globally...), don't think anything recent is very problematic, and I'm kinda stuck waiting on rules before I can do much more - so maybe that should be a subject for some meeting??

**Comment by mkoo** [comment]
Ugh I was hoping to keep kicking this down the road...

I believe we need to separate out the tables by scale-- for example:
- 7.5 minute USGS topoquads
- 15 minute USGS topoquads
- 100,000k USGS topoquads
- 250,000k USGS topoquads
- other 
Then we need to impose some naming rules so it will be easier to associate with the correct web service or USGS metadata (there is more than one option for the latter).

I think this is clean-up task and not really committee work-- since we can see the usage HG and adding topoquad to HG would break current guidelines and expectations. but let's discuss next week

**Comment by dustymc** [comment]
>kicking this down the road

Does not seem unreasonable, as far as I know these are primarily curatorial/not "research" data, so any denormalization probably has limited impacts. I'll adjust the priority accordingly.

>separate out the tables 

I don't have much of an opinion beyond the general feel that any increased complexity should pay for itself by DOING STUFF.

**Comment by Jegelewicz** [comment]
Rather than splitting tables, it might be better to just add some metadata to the current table.

quad | scale | map system | documentation | metadata 
 -- | -- | -- | -- | -- 
Pasadena Valley 7.5 minute [[ link ]](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctquad#pasadena_valley_7_5_minute) | 1:20,000 | US Topo | Pasadena Valley, Idaho, Elmore County | https://ngmdb.usgs.gov/ht-bin/tv_browse.pl?id=429a31d47cd1d6fcd2f15c8265a4c6f2 https://github.com/ArctosDB/arctos/issues/7121 | 
Horne Lake 92 F/7 [[ link ]](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctquad#horne_lake_92_f_7) | 1:50,000  | Canadian National Topographic System (NTS) | Canada, British Columbia | https://github.com/ArctosDB/code-table-work/issues/51 |

**Comment by dustymc** [comment]
>add some metadata 

That's not technically viable.

**Comment by Jegelewicz** [comment]
Are you saying that we cannot add any information to the code table that would allow us to filter or parse through the information?

I feel like the request to split this table up is primarily because people don't like looking through a giant list of things. If we added some information that let code table users filter, it might make the long list less onerous.

**Comment by dustymc** [comment]
@ArctosDB/paleontology this should be of some interest to ya'll as well.

I'm going to change the structure (https://github.com/ArctosDB/dev/issues/42) of ctquad very soon (tonight?) because the data (which contain malformed HTML) are breaking https://github.com/ArctosDB/dev/issues/460.

I'll update some stuff to fix the data problems, hopefully without mangling anything, but here's a backup just in case:

[ctquad.csv](https://github.com/user-attachments/files/25346026/ctquad.csv)

>add any information to the code table that would allow us to filter or parse 

We can do about anything as long as we maintain a unique index on the term.

**Comment by dustymc** [comment]
https://handbook.arctosdb.org/documentation/authorities.html#ctquad updated

data cleaned

unused data (and possible cleanup map in case of reload) unloaded to https://docs.google.com/spreadsheets/d/1mregvJQvI--vfOnbKGDEiwTrqmTZy7JpFop0w0GgVZg
